plugins/arDominionB5Plugin/scss/_fullwidthtreeview.scss

```diff
@@ -0,0 +1,78 @@
+@use "sass:math";
+
+// From css/fullWidthTreeView.css
+#fullwidth-treeview-header {
+    border: 1px solid #ddd;
+    border-bottom: none;
+    height: 20px;
+}
+
+#fullwidth-treeview-header div {
+    float: left;
+    background-color: #eee;
+    padding-left: 7px;
+}
+
+#fullwidth-treeview-row {
+    margin-bottom: 30px;
+    border: 1px solid #ddd;
+    clear: both;
+}
+
+#fullwidth-treeview {
+    overflow-x: hidden;
+    height: 100%;
+    background-color: #ffffff;
+}
+
+.ui-resizable-s {
+    background-image: url('/images/resizable_grip.png');
+    height: 9px;
+    border: 1px solid #ddd;
+    cursor: ns-resize;
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-color: #eee;
+    bottom: -11px;
+    left: -1px;
+    margin: -1px;
+}
+
+.fullwidth-treeview-toggle.active {
+    border: 1px inset #eee;
+}
+
+#fullwidth-treeview-loading {
+    float: right;
+    margin-right: 110px;
+    margin-top: 4px;
+}
+
+.ui-resizable-s:hover {
+    background-color: #ddd;
+};
+
+#treeview-menu li.disabled a {
+    color: #ddd;
+}
+
+#fullwidth-treeview-row a.jstree-anchor {
+    width: 96.5%;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.hierarchy > .container > input[type="button"], #main-column > input[type="button"] {
+    margin-top: 14px;
+    margin-left: 6px;
+    float: right;
+}
+
+// From search.less
+.full-treeview-section {
+    background-color: $gray-200;
+    border: 1px solid $black;
+    .actions {
+      margin-bottom: -20px;
+    }
+}
```

Comment:
I'd love to see if we can improve these styles, but no need to do it now as we chatted.

Reply:
👍